src/query/index.js

```diff
@@ -75,6 +80,7 @@ const decorateDetailBase = ({
   );
 };
 
+let pendingAutoFetch = false;
```

Comment:
why does this need to be set in the global scope?

Reply:
I tried to get this working a couple of other ways that didn't work and this did.  It's obviously not a long term solution but right now it's a blocker, and I feel like the long term solution requires an moving infinite scroll triggers to the application and out of qewl.

Given this solves the problem app-wide and hasn't broken on me yet once in 27 days, and will probably be removed as a feature entirely, eventually, I'm OK with the technical debt.